website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -31,7 +41,7 @@ To learn about the project directory structure, see [Kubebuilder project layout]
 
 `operator-sdk init` generates a `go.mod` file to be used with [Go modules][go_mod_wiki]. The `--repo=<path>` flag is required when creating a project outside of `$GOPATH/src`, as scaffolded files require a valid module path. Ensure you [activate module support][activate_modules] by running `export GO111MODULE=on` before using the SDK.
 
-### Manager
+## Manager
```

Comment:
This should remain under "Create a new project", as "Manager" doesn't fit the existing header semantics (steps in creating a project).
```suggestion
### Manager
```